### DISCUSSION TITLE
How can I automatically bring the Swipeable List back to its original position after clicking on the left or right buttons?

### DISCUSSION THREAD

**Discussion by JackTorresR** [react-native-elements/react-native-elements#3782] [Q&A]
I'm trying to implement a Swipeable List component in my React app, but I'm having trouble figuring out how to automatically reset its position to the center after the user clicks on the buttons that appear on the left or right side of each item. I've searched for solutions online, but so far nothing seems to work. Any help would be greatly appreciated.

**Comment by arpitBhalla** [comment]
Hii, you can do
```tsx
leftContent={(reset) => (
    <Button
      title="Info"
      onPress={() => reset()}
      icon={{ name: 'info', color: 'white' }}
      buttonStyle={{ minHeight: '100%' }}
    />
  )}
```

Refer [swipeable docs](https://reactnativeelements.com/docs/components/listitem_swipeable)
